src/services/item/test/index.test.ts

```diff
@@ -237,6 +238,23 @@ describe('Item routes tests', () => {
         expect(response.statusCode).toBe(StatusCodes.OK);
       });
 
+      it('Create successfully with description placement above and should not erase default thumbnail', async () => {
+        const payload = FolderItemFactory({
+          settings: { descriptionPlacement: DescriptionPlacement.ABOVE },
+        });
+        const response = await app.inject({
+          method: HttpMethod.Post,
+          url: `/items`,
+          payload,
+        });
+
+        const newItem = response.json();
+        expectItem(newItem, payload, actor);
+        expect(response.statusCode).toBe(StatusCodes.OK);
+        expect(newItem.settings.descriptionPlacement).toBe(DescriptionPlacement.ABOVE);
+        expect(newItem.settings.hasThumbnail).toBeFalsy();
```

Comment:
Are you checking `false` or `undefined`? `isFalsy` checks both, so it might be worth to use either `isUndefined` or `isEqual(false)`

Reply:
Ah okay, I will change that